brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/BrageType.java

```diff
@@ -66,7 +66,7 @@ public enum BrageType {
     FAGARTIKKEL("Fagartikkel"),
     EKSTERNNOTAT("Eksternnotat");
 
-    private static final String RAPPORT = "Rapport";
+    public static final String RAPPORT = "Rapport";
```

Comment:
why are RAPPORT and a few others just strings here, not enum?

Reply:
Looks like it is aliases for types.

Reply:
make sense :-)